### PR TITLE
Update docs

### PR DESCRIPTION
**Pull request by jwu910**
Updated docs for release preparation. Updated install instructions

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -0,0 +1,93 @@
...
+
+## Pull Request Process
+
+1. Ensure any install or build dependencies are removed before the end of the layer when doing a
+   build.
+2. Update the README.md with details of changes to the interface, this includes new environment
+   variables, exposed ports, useful file locations and container parameters.
+3. Increase the version numbers in any examples files and the README.md to the new version that this
+   Pull Request would represent. The versioning scheme we use is [SemVer](http://semver.org/).
+4. You may merge the Pull Request in once you have the sign-off of two other developers, or if you
```
Comment (rainrivas) [inline CONTRIBUTING.md]: Wouldn't it be best if we just approve/merge PRs for them. Contributors should really just make PRs and modify them based on comments. The merge itself should be controlled by repo owners to ensure merges are done the way we want them (squash/rebase/merge-commit etc)

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -0,0 +1,93 @@
...
+when an individual is representing the project or its community. Examples of
+representing a project or community include using an official project e-mail
+address, posting via an official social media account, or acting as an appointed
+representative at an online or offline event. Representation of a project may be
+further defined and clarified by project maintainers.
+
+### Enforcement
+
+Instances of abusive, harassing, or otherwise unacceptable behavior may be
+reported by contacting the project team at [INSERT EMAIL ADDRESS]. All
```
Comment (rainrivas) [inline CONTRIBUTING.md]: Should we come up with an email here, or should we just say "open an issue" until we set up an email address to reach people at?

Comment (jwu910) [reply CONTRIBUTING.md]: LOL My attention to detail was sub par. "[INSERT EMAIL ADDRESS]"

Let's take a look at the wording and we can take a look at the contributing docs.

File: CONTRIBUTING.md (hunk 1 above)
Comment (jwu910) [reply CONTRIBUTING.md]: Yeah, i wanted to talk about these. It was a generic template to follow. I can try to reword it to follow the workflow we've been using.

File: CONTRIBUTING.md (hunk 2 above)
Comment (rainrivas) [reply CONTRIBUTING.md]: @jwu910 how about "unacceptable behavior may be reported by contact the project team at their GH profiles" ? or we remove it for now until we come up with a unified shared account

Comment (jwu910) [reply CONTRIBUTING.md]: Report to the repo owner's email?

Comment (rainrivas) [reply CONTRIBUTING.md]: Yeah that could work

**Review by rainrivas — APPROVED** [review]
